src/test/java/org/opensearch/security/http/OnBehalfOfAuthenticatorTest.java

```diff
@@ -143,40 +140,13 @@ public void testBadKey() {
 
     @Test
     public void testWeakKeyExceptionHandling() throws Exception {
```

Comment:
@RyanL1997 If the tests pass can you review this change? I had to make quite a few changes to this test because the WeakKeyException was now being thrown [here](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/util/KeyUtils.java#L74). It looks like that call was not throwing the error previously.

Reply:
What I would like to verify is that using securityadmin.sh after changing the signing key to a weak key would result in a failure of the security admin script while the nodes remain up and with the previous good configuration.